src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -376,6 +380,23 @@ private void writeAlertToDB(Alert alert, HistoryReference ref) throws HttpMalfor
 
     }
 
+    private TreePath findAlertInTree(Alert alert) {

```

Comment:
Hmmm ok the updated alert always becomes the 0th item in the branch, so that makes sense. However, now I'm back to the item being selected but the selection not being displayed :(